examples/wasm/BUILD

```diff
@@ -8,10 +8,37 @@ licenses(["notice"])  # Apache 2
 
 envoy_package()
 
+filegroup(
+    name = "configs",
+    srcs = glob(
+            [
+                "**/*.wasm",
+            ],
+    ) + select({
+        "//bazel:linux_x86_64": glob(
```

Comment:
this passes `config_validation` on the `x64 release` run, but fails on `compile_time_options` - not sure why

Reply:
https://dev.azure.com/cncf/envoy/_build/results?buildId=54678&view=logs&j=e969334a-0e55-5c18-ac96-8b546753391e&t=b5329a9e-93c7-5412-9f4c-6f8d659e1b57&l=4144

Reply:
thinking this should actually work on win/mac also - just only on x86_64 arch

Reply:
ah, k - i see

not sure how to exclude this test

Reply:
i think this should be resolved now